tests/test_freshness_date_parser.py

```diff
@@ -1579,6 +1581,9 @@ def test_freshness_date_with_relative_base_past(self, date_string, date, time):
     @parameterized.expand([
         param('3 days', date(2010, 6, 7), time(13, 15)),
         param('2 years', date(2012, 6, 4), time(13, 15)),
+        param('next monday', date(2010, 6, 7), time(13, 15)),
+        param('next friday', date(2010, 6, 11), time(13, 15)),
+        param('next saturday', date(2010, 6, 5), time(13, 15))
```

Comment:
I miss:
- A test with just the weekday (if we have those somewhere else, why are these tests not in the same group of tests?)
- A test with `nextdoor` in it, verifying that `nextdoor` does not work the same as `next`.

Reply:
> * A test with just the weekday (if we have those somewhere else, why are these tests not in the same group of tests?)

cause `next` belongs to `freshness_date_parser.py` same as `ago`

Reply:
how to add tests which return None?

Reply:
I believe you can use `test_insane_dates` for that.

Reply:
Done